CMJS.cs

```diff
@@ -28,25 +28,45 @@ public class CMJS
     private BPMChangeGridContainer bpmEventsContainer;
     private List<Check> checks = new List<Check>()
     {
-        new EmptyCheck()
+        // Handle yourself...
     };
     private CheckResult errors;
     private UI ui;
     private AudioTimeSyncController atsc;
     private int index = 0;
     private bool movedAfterRun = false;
 
+    private const string JsScriptDirectory = "CM-JsScripts";
```

Comment:
```suggestion
    private const string JsScriptDirectory = "CM-JS-Scripts";
```
The plugin is `CM-JS` so lowercase feels off imo.

Also you probably want to put these constants somewhere or expose it so `ExternalJS.cs` can reference the same constant.